src/main/java/seedu/address/logic/pilot/deletepilot/DeletePilotCommand.java

```diff
@@ -34,7 +34,7 @@ public DeletePilotCommand(String uuid) {
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        deletedPilot = model.getPilotManager().getItem(uuid);
```

Comment:
Thanks for the rename.

Reply:
Actually, thinking about this, deleted pilot makes more sense, because it would be there for undo/redo features. Nevertheless, since we won't be implementing undo/redo in a short while, we can even remove this field. So naming probably wouldn't be that much of a deal.